packages/sns/src/types/ledger.responses.ts

```diff
@@ -0,0 +1,11 @@
+import type { Value } from "../../candid/icrc1_ledger";
+
+// Source: https://github.com/dfinity/icrc-1#standard-metadata-entries
+export enum SnsMetadataEntries {
+  SYMBOL = "icrc1:symbol",
+  NAME = "icrc1:name",
+  DECIMALS = "icrc1:decimals",
+  FEE = "icrc1:fee",
+}
+
+export type SnsTokenMetadata = [string | SnsMetadataEntries, Value][];
```

Comment:
Is the `string |` necessary here?

Reply:
Did not find a better solution because Candid uses `[string, Value]` so instead of having just `string` I thought adding it at least with the enum as well.

But really happy if you have a better solution!!!!

Reply:
Got it, I thought that there will be no other keys (https://github.com/dfinity/icrc-1#standard-metadata-entries). Unfortunately I see no better solution 🤷‍♂️